files/sharepointplus/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "sharepointplus",
+  "repo": "aymkdn/sharepointplus",
+  "files": {
+    "basePath": "jsdelivr/"
```

Comment:
There is no such folder here https://github.com/Aymkdn/SharepointPlus/tree/v3.12